app/controllers/projects/subscriptions_controller.rb

```diff
@@ -0,0 +1,51 @@
+class Projects::SubscriptionsController < ApplicationController
+  after_filter :verify_authorized, except: [ :update_status ]
+
+  def update_status
+    subscription = Subscription.find_by(subscription_code: params[:id])
+    RecurringContribution::UpdateStatus.process(request, subscription)
+    render nothing: true, status: :ok
+  rescue RecurringContribution::UpdateStatus::InvalidRequestError
+    render json: { error: 'invalid postback' }, status: :bad_request
+  end
+
+  def new
+    check_authorization_for_subscription_instance
+
+    render nothing: true
+  end
+
+  def create
+    check_authorization_for_subscription_instance
+
+    @subscription = RecurringContribution::Subscriptions::CreateJuntos.process(
```

Comment:
What do you thing if we move this guy to a private method?

```ruby
@subscription = process_subscrition

.....

private

def process_subscription
RecurringContribution::Subscriptions::CreateJuntos.process(
.....
end
```

`process_subscription` is just a example of name